06-Modelling-Rigid-Bodies/01-Introduction-to-Rigid-Bodies.ipynb

```diff
@@ -0,0 +1,839 @@
+{
```

Comment:
<p>you could use <code>numpy.full(N_particles, 2)</code></p><p> </p>
<br/>

 _Reply via <a href='https://app.reviewnb.com/glotzerlab/hoomd-examples/pull/61/discussion/'>ReviewNB</a>_ <div id='ReviewNBCommentContext-DoNotDelete' style='display:none' data-state='OPEN' data-cellIndex='19'/>

Reply:
<p>I could. I'll keep the list expansion for consistency and clarity for new users.</p>